packages/backend/src/app.ts

```diff
@@ -329,6 +338,23 @@ export class App {
               route = connectionRoutes.get
               router[method](
                 toRouterPath(path),
+                connectionMiddleware,
```

Comment:
this will be nicer once the magic for loop is gone (ie https://github.com/interledger/rafiki/issues/916)